crates/compiler/mono/src/ir/pattern.rs

```diff
@@ -1447,8 +1463,10 @@ fn store_list_pattern<'a>(
     layout_cache: &mut LayoutCache<'a>,
     list_sym: Symbol,
     list_arity: ListArity,
+    list_layout: InLayout<'a>,
     element_layout: InLayout<'a>,
     elements: &[Pattern<'a>],
+    opt_rest: &Option<(usize, Option<Symbol>)>,
```

Comment:
Is this ever (usize, None)?

Reply:
`(usize, None)` does happen higher up in the ast if we have a spread that doesn't capture (at least that is my assumption, didn't double check the code the code)

Originally I had it converting to `Option<(usize, Symbol)>` cause mono only cares about it if the symbol exists. 

I removed it cause it felt unnecessary to have two checks. One to go to this other format and one in mono to decide if we should generate the the binding for rest. I can change it back to that if you want.

Reply:
Ah gotcha. Makes sense. No need to change i think